crates/commonware-node/src/dkg/manager/ingress.rs

```diff
@@ -29,31 +29,28 @@ impl Mailbox {
     ///
     /// Returns `None` if this node was not a dealer, or if the request is
     /// for a different epoch than the ceremony that's currently running.
-    pub(crate) async fn get_intermediate_dealing(
+    pub(crate) async fn get_dealer_log(
```

Comment:
should we stick with `dealer log` instead of `intermediate dealing` nomenclature? in this method doc and other mentions